tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -69,8 +69,8 @@ jobs:
       displayName: 'Clean untagged docker images'
       inputs:
         script: |
-          docker rm $(docker ps -a | grep Exited | awk '{print $1;}') || true
-          docker images -q --filter "dangling=true" | xargs -n1 -r docker rmi
+          docker container prune -f
```

Comment:
Does this need `-a` option?

Reply:
I guess this change is keeping parity with the previous change where "dangling=true" was specified.

Reply:
Yes, `-a` seems overkill to me.

Reply:
Yes, the `-a` seems overkill to me.